src/generic_sort.cpp

```diff
@@ -17,7 +17,6 @@
 
 #include <functional>
 #include <unordered_map>
-
```

Comment:
Was this change done by ClangFormat? If not, I'd prefer to exclude it from this PR.